pkg/util/metric/mometric/metric.go

```diff
@@ -115,21 +117,46 @@ func InitMetric(ctx context.Context, ieFactory func() ie.InternalExecutor, SV *c
 	logutil.Infof("metric with UpdateStorageUsageInterval: %v", initOpts.updateInterval)
 }
 
+func registerFileService(ctx context.Context, nodeUUID, role string, initOpts InitOptions) {
+	switch strings.ToUpper(role) {
+	case "CN", "DN":
+		registry := prom.NewRegistry()
+		for _, c := range metric.FSCollectors {
+			registry.MustRegister(c)
+		}
+		c := newMetricLogCollector(WithFlushInterval(3 * time.Second))
+		e := newMetricExporter(registry, c, nodeUUID, role, initOpts.metricLogGatherInterval /* new config only for MetricLog */)
```

Comment:
diff exporter needs diff gatherInterval, so it need new config like `metricLogGatherInterval` for MetricLog